src/dotnet/commands/dotnet-install/dotnet-install-tool/ProjectRestorer.cs

```diff
@@ -69,5 +69,28 @@ public ProjectRestorer(IReporter reporter = null)
                 throw new ToolPackageException(LocalizableStrings.ToolInstallationRestoreFailed);
             }
         }
+
+        private static void WriteLine(IReporter reporter, string line, FilePath project)
+        {
+            line = line ?? "";
+
+            // Remove the temp project prefix if present
+            if (line.StartsWith($"{project.Value} : ", StringComparison.CurrentCultureIgnoreCase))
+            {
+                line = line.Substring(project.Value.Length + 3);
```

Comment:
This math may be wrong for some cultures.

Reply:
I assume you mean the `project.Value.Length` and this should use a `StringInfo` to arrive at the correct length?

Reply:
Well, I mean if we need to compare to project.Value by CurrentCulture. If we can check for project.Value at start with Ordinal, then this is fine. It is related to my other comment.

Reply:
Do we have full control over project.Value? Will it have characters outside the ASCII range. Who puts that ' : ' is it also not localized like 'error' and 'warning'?

Reply:
We do have control over the project path, but it is based on `Path.GetTempPath`.

The ` : ` comes from the format MSBuild uses for its console diagnostic messages.  Likewise, `warning` and `error` are intentionally not localized, presumably so that the level can be parsed in a manner like what is being done here.

Reply:
I think we can just use OrdinalIgnoreCase, then.

Reply:
I'm fine with `OrdinalIgnoreCase` as a safe-guard (I suspect `Ordinal` would work too provided MSBuild doesn't try to normalize the path given).

Reply:
Changed to `OrdinalIgnoreCase`.

Reply:
👍